celsius_convert.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby 
+
+print "Hello. Please enter a Celsius value: "
+celsius = gets.chomp
+begin
+    celsius = Integer(celsius)
+fahrenheit = (celsius * 9 / 5) + 32
+puts "Saving result to output file 'temp.out'"
+    begin    
```

Comment:
Don't nest exception handlers. Do it like so:
```ruby
begin
  # statement that might throw exception A
  # more code
  # statement that might throw exception B
rescue ExceptionA, ExceptionB
  # ...
end
```

Or

```ruby
begin
  # statement that might throw exception A
  # more code
  # statement that might throw exception B
rescue ExceptionA
  # ...
rescue ExceptionB
  # ...
end
```

Reply:
Look up what specific exceptions are thrown when
1. You perform arithmetic on a string, and B
2. There's a file I/O error